app/assets/javascripts/application/evaluations.js.coffee

```diff
@@ -0,0 +1,34 @@
+#= require 'select2.min'
```

Comment:
I'm not working with javascript classes, mostly. But it's a good idea to use it here. I'll update it today's evening.